ansible/avena/roles/can/tasks/main.yml

```
@@ -0,0 +1,68 @@
+- name: ensure CAN interfaces
+  when: oats_can_interfaces is defined
+  block:
+    - name: ensure can@ service is installed
+      become: yes
+      copy:
+        dest: /etc/systemd/system/can@.service
+        src: systemd/can@.service
+        owner: root
+        group: root
+        mode: "0644"
+      notify: reload systemd
+
+    - name: ensure can interfaces are enabled and configured
+      become: yes
+      copy:
+        dest: /etc/udev/rules.d/
+        src: udev/60-can.rules
+        owner: root
+        group: root
+        mode: "0644"
+      notify: reload udev
+
+    - name: ensure can interfaces are started
+      become: yes
+      systemd:
+        name: can@{{ item }}
+        state: started
+      with_items: "{{ oats_can_interfaces }}"
+
+    # TODO: This is just an apalis issue?
+    - name: ensure can interfaces are reset after suspend (wakeup issues)
+      become: yes
+      template:
+        src: system-sleep/reset-can-interfaces.j2
+        dest: /usr/lib/systemd/system-sleep/reset-can-interfaces
+        owner: root
+        group: root
+        mode: "0755"
+
+    - name: Can watchdog
+      block:
+        - name: ensure can-watchdog.service is installed
+          become: yes
+          copy:
+            dest: /etc/systemd/system/can-watchdog.service
+            src: systemd/can-watchdog.service
+            owner: root
+            group: root
+            mode: "0644"
+          notify: reload systemd
+
+        - name: ensure can_watchdog is installed
+          become: yes
+          copy:
+            dest: /usr/local/bin
+            src: can_watchdog
+            owner: root
+            group: root
+            mode: "0700"
+
+        - name: ensure can-watchdog.service is enabled
+          become: yes
+          systemd:
+            name: can-watchdog
+            daemon_reload: yes
+            enabled: "{{ oats_can_watchdog }}"
+            state: "{{ 'stopped' if not oats_can_watchdog | default('started') }}"
```

This still needs to be fixed. For the jvk deployment I used 
```
state: "{{ 'stopped' if not oats_can_watchdog else 'started' }}"
```
However that does not work if `oats_can_watchdog` is undefined and reads awkwardly. Maybe `'started' if oats_can_watchdog is 'yes' else 'stopped'`? Have not tested it to ensure it handles undefineds properly

```suggestion
            state: "{{ 'started' if oats_can_watchdog is 'yes' else 'stopped' }}"
```

Relevant documentation on inline ifs in jinja2: https://jinja.palletsprojects.com/en/2.11.x/templates/#if-expression